openapi_core/wrappers.py

```diff
@@ -107,6 +110,73 @@ def mimetype(self):
         return self.request.mimetype
 
 
+class RequestsOpenAPIRequest(BaseOpenAPIRequest):
+
+    def __init__(self, response, path_pattern, regex_pattern, server_pattern):
```

Comment:
I don't think the requests library explicitly uses a request object in its normal flow. Instead the response starts out as a function. You don't get an actual response object until after the http result is returned.
`
response = requests.post('http://httpbin.org/post', data = {'key':'value'})
request = response.request
`

Reply:
Ah, I misread what you were getting at. I pass in the response because of the way that the requests library distributes its data. For example, in order to get cookies it needs the response object even though cookies are a part of the request model in your interface.